client/my-sites/plugins/plugin-information/index.jsx

```diff
@@ -147,6 +148,19 @@ export default React.createClass( {
 		return {};
 	},
 
+	getDefaultActionLinks( plugin ) {
+		let adminUrl = get( this.props, 'selectedSite.options.admin_url' );
```

Comment:
Hmm, too bad this component isn't `connect()`ed. Would be much nicer to use the `getSiteAdminUrl()` selector instead 😕